_includes/misc/supported_versions.md

```diff
@@ -4,6 +4,7 @@ Supported versions of {{site.productname}}:
 
 | Version |   Release Date    |  End of Support   |
 |:-------:|:-----------------:|:-----------------:|
+|   5.5   | TBD               |        TBD        |
```

Comment:
Why not wait until we know the release date for 5.5 before adding this line?

Reply:
I've added this here so I don't forget. I _should_ spot the TBD when I do a final review of the release PR.